containers/elixir-phoenix-postgres/README.md

```diff
@@ -0,0 +1,71 @@
+# Ruby
+
+## Summary
+
+_Develop Elixir/Phoenix based applications. includes everything you need to get up and running._
+
+| Metadata                    | Value                 |
+| --------------------------- | --------------------- |
+| _Contributors_              | [idyll]               |
+| _Definition type_           | Dockerfile            |
+| _Works in Codespaces_       | Yes                   |
+| _Container host OS support_ | Linux, macOS, Windows |
+| _Languages, platforms_      | Elixir                |
+
+## Using this definition with an existing folder
+
+While this definition should work unmodified, you can select the version of Elixir the container uses by updating the `VARIANT` arg in the included `devcontainer.json` (and rebuilding if you've already created the container).
+
+```json
+"args": { "VARIANT": "1.10" }
```

Comment:
You'll need to do this in `docker-compose.yml` as mentioned in other comments. Args in devcontainer.json is only for simple single container scenarios.